frontend/src/bundles/common/components/side-bar/side-bar.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import {
+    Box,
+    Flex,
+    Icon,
+    IconButton,
+    IconMap,
+    Spacer,
+} from '~/bundles/common/components/components.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { Size } from '../icon/icon.js';
+import {
+    CollapseButton,
+    CollapseButtonLink,
+} from './libs/components/components.js';
+
+const SideBar = (): JSX.Element => {
+    const [isCollapsed, setIsCollapsed] = useState(false);
+    const { pathname } = useLocation();
+    const navigate = useNavigate();
+
+    const toggle = useCallback(
+        (): void => setIsCollapsed(!isCollapsed),
+        [isCollapsed],
+    );
+
+    const activeIconPage = (page: ValueOf<typeof AppRoute>): string => {
+        return pathname === page ? 'white' : 'background.600';
+    };
+
+    const logOut = useCallback(() => {
+        //ToDo: log out user with token
+        navigate(AppRoute.SIGN_IN);
+    }, [navigate]);
+
+    return (
+        <Flex
+            w={isCollapsed ? '60px' : '260px'}
+            bg="background.900"
+            height="100vh"
+            position="fixed"
+            flexDirection="column"
+            justifyContent="space-between"
+            p="10px"
+            pb="20px"
+        >
+            <IconButton
+                bg="none"
+                aria-label={isCollapsed ? 'expand' : 'collapse'}
+                icon={
+                    isCollapsed ? (
+                        <Icon as={IconMap.ARROW_RIGHT} />
+                    ) : (
+                        <Icon as={IconMap.ARROW_LEFT} />
+                    )
+                }
+                mb="10px"
+                onClick={toggle}
+                _active={{ bg: 'none' }}
+                _hover={{ bg: 'none' }}
+                justifyContent={isCollapsed ? 'center' : 'flex-end'}
+            />
```

Comment:
Maybe, it is better to create a custom variant for this IconButton so we can later reuse it?